sympy/matrices/tests/test_densearith.py

```diff
@@ -52,3 +54,7 @@ def test_mulmatscaler():
 
     assert mulmatscaler(a, ZZ(4), ZZ) == [[ZZ(4), ZZ(0), ZZ(0)], [ZZ(0), ZZ(4), ZZ(0)], [ZZ(0), ZZ(0), ZZ(4)]]
     assert mulmatscaler(b, ZZ(1), ZZ) == [[ZZ(3), ZZ(7), ZZ(4)], [ZZ(2), ZZ(4), ZZ(5)], [ZZ(6), ZZ(2), ZZ(3)]]
+
+def test_eq():
+    assert Matrix([[1]]) != object() #Earlier gave Syntax and SympifyError , now returns False
```

Comment:
These comments seem unnecessary to me.

Reply:
I'll remove them

Reply:
If you want to comment, you can reword it to make them more clear. But I think it is better to remove them.

Reply:
Sure